BXMSDOC-5711: Release notes docs for DM and PAM 7.8 (#79)
kaldesai/kie-docs: doc-content/enterprise-only/release-notes/rn-780-fixed-issues-ref.adoc

[id='rn-770-fixed-issues-ref']
= Fixed issues in {PRODUCT} {ENTERPRISE_VERSION_LONG}

{PRODUCT} {ENTERPRISE_VERSION_LONG} provides increased stability and fixed issues listed in this section.

== {CENTRAL}

ifdef::PAM[]

* Dashbuilder data transfer feature does not work on Windows [https://issues.redhat.com/browse/RHPAM-2751[RHPAM-2751]]

endif::[]

* In {CENTRAL}, you can not re-enable the clone project using http [https://issues.redhat.com/browse/RHPAM-2721[RHPAM-2721]]

> * An unexpected conflicting rows error occurs when you try to modify guided decision table [https://issues.redhat.com/browse/RHPAM-2673[RHPAM-2673]]

Reviewer: ```suggestion
* An unexpected conflicting row error occurs when you try to modify a guided decision table [https://issues.redhat.com/browse/RHPAM-2673[RHPAM-2673]]
```

ifdef::PAM[]
== Process engine

* Expression in timer is not interpreted correctly when `"` is used for literal expression [https://issues.redhat.com/browse/RHPAM-2845[RHPAM-2845]]

== Process Designer

* In {CENTRAL}, if you try to use timer expression components, you receive an error message [https://issues.redhat.com/browse/RHPAM-2695[RHPAM-2695]]
* You cannot remove the case file and global variables [https://issues.redhat.com/browse/RHPAM-2643[RHPAM-2643]]
* In the process designer, if you set an incorrect `calledElement` attribute, it breaks the process and you receive an error message [https://issues.redhat.com/browse/RHPAM-2432[RHPAM-2432]]

endif::[]

ifdef::DM[]

== Decision engine
* In DMN model, you can generate a strongly typed code for DMNContext and DMNResult with the help of codegen facility [https://issues.redhat.com/browse/RHDM-1323[RHDM-1323]]

endif::[]

== {OPENSHIFT}

> * You can not deploy immutable {KIE_SERVER} environments on {OPENSHIFT} using Operators [https://issues.redhat.com/browse/RHPAM-2942[RHPAM-2942]]

Reviewer: ```suggestion
* You cannot deploy immutable {KIE_SERVER} environments on {OPENSHIFT} using Operators [https://issues.redhat.com/browse/RHPAM-2942[RHPAM-2942]]
```

ifdef::PAM[]

> * You can not create custom extension image for database [https://issues.redhat.com/browse/RHPAM-2948[RHPAM-2948]]

Reviewer: cannot should be used as one word as per IBM SG.
```suggestion
* You cannot create a custom extension image for database [https://issues.redhat.com/browse/RHPAM-2948[RHPAM-2948]]
```
